sktime/forecasting/base/_meta.py

```diff
@@ -25,10 +26,14 @@ class _HeterogenousEnsembleForecaster(_HeterogenousMetaEstimator, BaseForecaster
     # this must be an iterable of (name: str, estimator, ...) tuples for the default
     _steps_fitted_attr = "forecasters_"
 
-    def __init__(self, forecasters, n_jobs=None, fc_alt=None):
+    def __init__(
+        self, forecasters, backend=None, backend_params=None, n_jobs=None, fc_alt=None
```

Comment:
I recommend, put the new parameters at the end, to avoid breaking downward scompatibility.